docs/concepts/withdrawals.md

```diff
@@ -9,8 +9,8 @@ sidebar_position: 8
 Validators staking ether on Mainnet after [The Merge](merge.md), accrue two
 forms of rewards:
 
-- Execution layer rewards paid directly to a withdrawal address (Ethereum address).
-- Consensus layer rewards for performing actions each epoch.
+- execution layer rewards paid directly to a withdrawal address (Ethereum address).
+- consensus layer rewards for performing actions each epoch.
```

Comment:
I think since they're fragments continuing from the introductory clause, they start with lowercase letters and are separated by commas, with a period at the end of the last bullet.

Reply:
Please review the list guidelines from the Microsoft Style Guide: https://learn.microsoft.com/en-us/style-guide/scannable-content/lists. Items should always start with an uppercase letter, even if they're fragments, unless they start with a technical term that is always lowercase.